src/pages/Flow/FlowRunTable-Tile.vue

```diff
@@ -46,7 +46,30 @@ export default {
       page: 1,
       searchTerm: null,
       sortBy: 'scheduled_start_time',
-      sortDesc: true
+      sortDesc: true,
+      state: [],
+      states: [
```

Comment:
We should sort these here and on the task run tile; since there's no text input it's a little difficult atm to find the one you're looking for